hooks/post/30-upgrade.rb

```diff
@@ -51,7 +51,7 @@ def import_puppet_modules
 end
 
 def import_subscriptions
```

Comment:
I think you can remove this one since it's duplicate for older users. Users who already ran it won't have problems anyway.

Reply:
@ekohl @ehelms is there a consensus? wondering what to do about this PR

Reply:
@ehelms since the 3.4 upgrade will execute all these steps, won't that essentially be the same as including 3.3?

Reply:
@ehelms ping re ^

Reply:
Yes - assuming they are the same task.